### ISSUE TITLE
saving/submitting already submitted records

### ISSUE THREAD

**Issue by Mbrownshoes** [cioos-siooc/metadata-entry-form#129]
seems a bit odd that if I edit an already submitted record, the submit section is greyed out, but any saves update the record..

how about if you save a change made on a submitted record, it changes the record to draftand then you need to hit the submit button to actually submit?

at the very least the fix could involve telling the user that any saved changes to submitted form are auto-submitted

**Comment by n-a-t-e** [comment]
I agree that this this needs to be changed. Otherwise records can get into a weird state where the record is submitted but it's not necessarily complete, if a field was removed after being submitted.

_how about if you save a change made on a submitted record, it changes the record to draftand then you need to hit the submit button to actually submit?_

This sounds like a good solution
